string[] joint_names
geometry_msgs/Point[] joints